src/Accordion/Accordion.tsx

```diff
@@ -46,7 +46,7 @@ export class Accordion extends React.Component<AccordionProps, AccordionState> {
                 {this.props.list.map((item: AccrodionListItem, index: number) =>
```

Comment:
this will crash when the `list` prop is accidentally set to `null`. Use this instead `this.props.list && this.props.list.map`

Reply:
resolved in commit https://github.com/sebgroup/react-components/pull/27/commits/82e4778e78a4c6f887280eb5c2db59b485b9c99f